programs/wen_new_standard/src/instructions/mint/create.rs

```diff
@@ -49,6 +50,7 @@ pub struct CreateMintAccount<'info> {
         extensions::metadata_pointer::metadata_address = mint,
         extensions::group_member_pointer::authority = manager,
         extensions::transfer_hook::authority = authority,
+        extensions::permanent_delegate::delegate = args.permanent_delegate.unwrap_or_else(|| manager.key()),
```

Comment:
I dont think we should leave permanent delegate as manager if user chose not to use it, can you add an update permanent delegate below to remove it if the argument passed is None